main/src/cgeo/geocaching/models/Trackable.java

```diff
@@ -214,6 +220,35 @@ public void setReleased(@Nullable final Date released) {
         this.released = released == null ? null : new Date(released.getTime()); // avoid storing external reference in this object
     }
 
+    @Nullable
+    public Date getLogDate() {
+        if (logDate != null) {
+            return new Date(logDate.getTime());
+        }
+        return null;
+    }
+
+    public void setLogDate(@Nullable final Date logDate) {
+        // avoid storing external reference in this object
+        this.logDate = logDate != null ? new Date(logDate.getTime()) : null;
+    }
+
+    public String getLogType() {
+        return logType;
+    }
+
+    public void setLogType(String logType) {
```

Comment:
There is already an enum for Logtypes of Trackables. Did you consider using it and decided against?